hideShowPassword.js

```diff
@@ -52,6 +52,11 @@
     // set after input has been inserted into the DOM.
     enable: canSetInputAttribute,
 
+    // Event to trigger whenever the element is toggled.
+    // For example, if 'focus' it will focus the cursor in the
+    // input element after toggling.
+    triggerOnToggle: '',
```

Comment:
Updated PR with a default `triggerOnToggle: false` and a `boolean` check before calling the event.  I'm not 100% sure on the performance either, but I assume that a boolean check is cheaper than an empty event call.